### ISSUE TITLE
Django >= 2.0 not supported

### ISSUE THREAD

**Issue by paddatrapper** [revsys/django-tos#34]
The ondelete argument is a required argument from Django 2.0 - see [the docs](https://docs.djangoproject.com/en/1.10/ref/models/fields/#foreignkey). Please add support for Django 2.0 and higher

```
File "/seed/src/django-tos-master/tos/models.py", line 71, in UserAgreement
  terms_of_service = models.ForeignKey(TermsOfService, related_name='terms')
TypeError: __init__() missing 1 required positional argument: 'on_delete'
```

**Comment by paddatrapper** [comment]
PR: #32

**Comment by nicholasserra** [comment]
Tracking in https://github.com/revsys/django-tos/pull/35

**Comment by nicholasserra** [comment]
@paddatrapper wanna give my PR here a try? I haven't ran this project in a while and wanna make sure this is working as expected before I merge and cut a new release

https://github.com/revsys/django-tos/pull/35

**Comment by paddatrapper** [comment]
Thanks! Sure will do

**Comment by nicholasserra** [comment]
Fixed in #35
